src/backend/binaryen/wasm_expr_gen.ts

```diff
@@ -80,6 +80,7 @@ import {
     EnumerateKeysGetValue,
     VTableGetValue,
     VTableSetValue,
+    NativeFunctionCallValue,
```

Comment:
Why we need to add such a semantic value? I don't think it's reasonable in semantic tree layer.
In my understanding, we just need to call the `dyntype_get_keys` API when processing `enumerate_keys` node